### PR TITLE
Materialize the window_id and session_id on the events table

### PR DESCRIPTION
**Pull request by rcmarron**
## Changes

Materializes  the `window_id` and `session_id` on the events table.

These properties are used to link the events to recordings, so by materializing them, we speed up the persons modal queries that rely on them.

## How did you test this code?

Added tests + verified things still worked

### CODE REVIEW

**Comment by neilkakkar** [comment]
(a benchmark test would be ideal here)

**Comment by rcmarron** [comment]
> (a benchmark test would be ideal here)

Agreed! The issue is that in the benchmark server, the event + session_recording_event data doesn't overlap, so it can't really measure the insight<>recording queries well.

@macobo and I had talked about this ~ a month ago, and decided it was best to keep the feature behind a FF and "benchmark against prod".

To add a bit of confidence to the change, I ran some tests in metabase, and if we run the persons query as it is today, but make the change to extract these 2 properties from the events table (which the recordings<>insights feature needs), it slows the query down by 20x in some cases - hence the call to materialize them.

**Comment by rcmarron** [comment]
@fuziontech this is going to run a migration on the events table - I'd love to get your eyes on it, and will definitely keep you in the loop before merging. 

Also, I assume we're good to run events table migrations again?

**Review by mariusandra — APPROVED** [review]
Since a review was requested, I can say that this looks good to me. 

Ran locally, all seemed fine, but I don't have a mental map of the nitty-gritty here, so I can't guarantee I didn't miss anything.

**Review by neilkakkar — APPROVED** [review]
LGTM!

One open question: Not everyone uses recordings. So, is there a good reason for having these materialised for everyone, vs, letting `materialize_properties_task` figure this out, and materialise these columns for people actually using the modals / link to recordings?

**Comment by mariusandra** [comment]
Recordings can be toggled on/off at will, so best to have this in I think.

**Review by fuziontech — APPROVED** [review]
🚢 it

File: ee/clickhouse/sql/events.py [hunk 1]
```diff
@@ -31,6 +30,9 @@
     , $group_2 VARCHAR materialized trim(BOTH '\"' FROM JSONExtractRaw(properties, '$group_2')) COMMENT 'column_materializer::$group_2'
     , $group_3 VARCHAR materialized trim(BOTH '\"' FROM JSONExtractRaw(properties, '$group_3')) COMMENT 'column_materializer::$group_3'
     , $group_4 VARCHAR materialized trim(BOTH '\"' FROM JSONExtractRaw(properties, '$group_4')) COMMENT 'column_materializer::$group_4'
+    , $window_id VARCHAR materialized trim(BOTH '\"' FROM JSONExtractRaw(properties, '$window_id')) COMMENT 'column_materializer::$window_id'
```
Comment (macobo) [inline ee/clickhouse/sql/events.py:33]: There was no need for this - last migration already creates this.

**Pull request merged**